src/gui/tray/CustomButton.qml

```diff
@@ -0,0 +1,68 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+
+Button {
+    id: root
+
+    property string imageSource: ""
+    property string imageSourceHover: ""
+
+    property string toolTipText: ""
+
+    property color textColor: ({})
+    property color textColorHovered: ({})
+
+    property color bgColor: "transparent"
+
+    property bool bold: false
+
+    Accessible.role: Accessible.Button
+    Accessible.name: text !== "" ? text : (toolTipText !== "" ? toolTipText : qsTr("Activity action button"))
+    Accessible.onPressAction: clicked()
```

Comment:
```suggestion
    Accessible.name: text !== "" ? text : (toolTipText !== "" ? toolTipText : qsTr("Activity action button"))
```

Since you are now inheriting from Button, this is already set.